lib/Db/AppointmentConfig.php

```diff
@@ -37,8 +37,8 @@
  * @method void setToken(string $token)
  * @method string getName()
  * @method void setName(string $name)
- * @method string getDescription()
- * @method void setDescription(string $name)
+ * @method string|null getDescription()
+ * @method void setDescription(?string $name)
```

Comment:
```suggestion
 * @method void setDescription(?string $description)
```

copy pasta from the first setter all the way down to the last ;-)